BP/common/culture/cultures/00_east_african.txt

```diff
@@ -0,0 +1,347 @@
+﻿east_african_group = {
```

Comment:
Isn't the Nilotic culture missing?

Reply:
They weren't used VIET cultures so I left them out as last time I asked that's what you said I should always do.

Reply:
Ok